jersey-servers/src/main/java/com/palantir/remoting2/servers/jersey/RemoteExceptionMapper.java

```diff
@@ -38,10 +38,23 @@
     @Override
     public Response toResponse(RemoteException exception) {
         String errorId = UUID.randomUUID().toString();
-        log.error("Error handling request {}", SafeArg.of("errorId", errorId), exception);
 
-        SerializableError error = exception.getRemoteException();
         Status status = Status.fromStatusCode(exception.getStatus());
+
+        // here in the client, log responses that indicate client error (4xx) at higher level than server error (5xx)
```

Comment:
well, it's in a server that gets an exception from another server. in that sense it's both in a client and in a server :)